resources/metadata/apiKeyValueMapUpload.yaml

```diff
@@ -0,0 +1,49 @@
+metadata:
+  name: apiKeyValueMapUpload
+  description: Upload an API key value map artifact into the API Portal
+  longDescription: |
+    With this step, you can upload an API key value map artifact to the API Portal using the OData API.
```

Comment:
Proposal: make clear it is the creation only.

Reply:
done